### ISSUE TITLE
If we create a book in gitbook ?

### ISSUE THREAD

**Issue by MoiRouhs** [monkstone/JRubyArtBook#1]
I will create the documentation in Spanish and French in gitbook, but  this moment I study, because I cann't write  more information about JRubyArt

**Comment by monkstone** [comment]
Yes this looks a reasonable approach, I already have calibre installed to do conversion to epub and or mobi which I was thinking of targeting. At the moment I am happiest with github flavored markdown, but I wondered whether I might want features from asciidoc. I think I will probably start in markdown (content will initially be more important than format).
PS: I can't see any money in it so intend keeping to open source.
